# Format is key: [value1, value2, ...] where key is a Python module and values are methods.
# Typically, the keys will start with "tests/java"
# Example: tests/java/whatever/testfile1.py: [method1, method7]

#We need an oracle license before this test can be hooked up
tests/java/functionality/basic_features/db.py: [test_select_oracle]

# test fails due to a test problem:
#INFO: Deploying web application directory /home/jenkins/apps/apache-tomcat-7.0.28/webapps/petclinic-0.1
#2014-02-25 17:29:39,257 [localhost-startStop-1] ERROR context.ContextLoader  - Context initialization failed
#org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'pluginManager' defined in ServletContext resource [/WEB-INF/applicationContext.xml]: Invocation of init method failed; nested exception is org.codehaus.groovy.grails.exceptions.NewInstanceCreationException: Could not create a new instance of class [HibernateGrailsPlugin]!
#        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:441)
#        at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
#        at java.util.concurrent.FutureTask.run(FutureTask.java:138)
#        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
#        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
#        at java.lang.Thread.run(Thread.java:662)
#Caused by: org.codehaus.groovy.grails.exceptions.NewInstanceCreationException: Could not create a new instance of class [HibernateGrailsPlugin]!
#        ... 6 more
#Caused by: java.lang.NoClassDefFoundError: org.hibernate.cfg.Environment
#        at java.lang.Class.forName0(Native Method)
#        at java.lang.Class.forName(Class.java:169)
#        at HibernateGrailsPlugin.class$(HibernateGrailsPlugin.groovy)
#        at HibernateGrailsPlugin.$get$$class$org$codehaus$groovy$grails$plugins$orm$hibernate$HibernatePluginSupport(HibernateGrailsPlugin.groovy)
#        at HibernateGrailsPlugin.<init>(HibernateGrailsPlugin.groovy:50)
#        ... 6 more
#Feb 25, 2014 5:29:39 PM org.apache.catalina.core.StandardContext startInternal
#SEVERE: Error listenerStart
#Feb 25, 2014 5:29:39 PM org.apache.catalina.core.StandardContext startInternal
#SEVERE: Context [/petclinic-0.1] startup failed due to previous errors
tests/java/functionality/framework/grails/grails_1.py: [test_grails_1_3_4]


# test_transaction_trace intermittently fails because we can't force the agent to report up the desired TT.
#  If the agent reports it, the test passes.  If the agent doesn't report it, the test fails.
# test_mobile_apm is correctly failing because of an agent problem
tests/java/functionality/server/resin.py: [test_transaction_trace, test_mobile_apm]


# performance tests are less important than functional tests now
tests/java/performance/metrics/play2_timeit.py: [test_disabled, test_enabled, test_enabled_tracer_naming_diabled]
